National Archives of Australia.js

```diff
@@ -186,7 +186,7 @@ function scrapeItem(doc, url) {
 	item.archiveLocation = meta.citation.replace(/^NAA\s*:\s*/i, '');
 	
 	var barcode = encodeURIComponent(meta['item barcode']);
-	item.url = 'http://www.naa.gov.au/cgi-bin/Search?O=I&Number=' + barcode;
+	item.url = 'http://recordsearch.naa.gov.au/scripts/AutoSearch.asp?O=I&Number=' + barcode;
```

Comment:
It looks like this site supports (and in fact redirects to) HTTPS, so these should all be changed to that.

Reply:
Done